src/components/Nav/Nav.scss

```
@@ -0,0 +1,99 @@
+.header {
+  display: flex;
+  min-width: 1260px;
+  height: 158px;
+  border-bottom: 1px solid #ededed;
+  .inner {
+    display: flex;
+    justify-content: space-between;
+    box-sizing: border-box;
+    position: sticky;
+    width: 1240px;
+    top: 0;
+    align-items: center;
+    margin: 0 auto;
+  }
```

```suggestion
.header {
  display: flex;
  min-width: 1260px;
  height: 158px;
  border-bottom: 1px solid #ededed;
  
  .inner {
    display: flex;
    justify-content: space-between;
    box-sizing: border-box;
    position: sticky;
    width: 1240px;
    top: 0;
    align-items: center;
    margin: 0 auto;
  }
```
선택자 사이는 가독성을 위해서 한칸씩 띄워주세요